.azure-pipelines/hpu-tests.yml

```diff
@@ -31,3 +31,22 @@ jobs:
         apt-get install -y hwinfo
         hwinfo --short
       displayName: 'Instance HW info'
+
+    - bash: |
+        pip install . --requirement requirements/test.txt
+      displayName: 'Install dependencies'
+
+    - bash: |
+         python ".azure-pipelines/run_hpu_tests.py"
+      displayName: 'HPU Tests in parallel'
+
+    # - bash: |
+    #     python "pl_examples/hpu_examples/simple_mnist/mnist.py"
+    #   displayName: 'Testing: examples'
```

Comment:
Is there a reason that this is commented out? Maybe I'm missing some context...

Reply:
Testing the pipeline out.